web/src/ShareSnapshotModal.tsx

```diff
@@ -275,7 +275,7 @@ const CodeSnippet = styled.code`
 `
 
 function downloadSnapshot(snapshot: Proto.webviewSnapshot) {
-  const timestamp = moment().format("YYYY-MM-DD_HHmmss")
+  const timestamp = snapshot.createdAt ?? moment().format("YYYY-MM-DD_HHmmss")
```

Comment:
`snapshot.createdAt` is going to be a full ISO8601 string, so I think it should be passed _to_ `moment()` so we always format or we'll get filenames like `tilt-snapshot_2022-05-11T20:10:17.882742.json` instead of `tilt-snapshot_2022-05-11_201017.json`

e.g.
```javascript
const createdAt = snapshot.createdAt ? moment(snapshot.createdAt) : moment();
const timestamp = createdAt.format("YYYY-MM-DD_HHmmss")
```
(`moment()` is cranky and doesn't like `null` or `""`)

Reply:
ah, good point! i've fixed the formatting.